src/Service/TransactionService.php

```diff
@@ -111,6 +116,19 @@ public function processTransaction($apiPayment)
 
             throw new TransactionException('Transaction failed', HttpStatusCode::HTTP_BAD_REQUEST);
         }
+
+         if (!empty($apiPayment->description)) {
+             $orderDescription = $apiPayment->description;
```

Comment:
$orderDescription = $apiPayment->description ?: $apiPayment->orderNumber;

maybe a bit less verbose.

Reply:
`$orderDescription = !empty($apiPayment->description) ? $apiPayment->description : $apiPayment->orderNumber;`

Your example is a bit wrong. $apiPayment->description ?? $apiPayment->orderNumber; would work, but it is from higher PHP versions.